resources/web/wwi/Parser.js

```diff
@@ -471,29 +471,19 @@ export default class Parser {
     let geometry;
     let appearance;
 
-    for (let i = 0; i < node.childNodes.length; i++) {
+    for (let i = node.childNodes.length - 1; i >= 0; i--) {
```

Comment:
more of a question, I've noticed you've reversed the order here. So the order in which children are provided matters? I must have picked the right one by chance with ColladaShape then

Reply:
No the order does normally not matter.

I reversed the order because the old animation were build like:
```
<Shape>
  <Appearance> </Appearance>
  <PBRAppearance> </PBRAppearance>
  <Geometry></Geometry>
</Ahape>
```

If someone try to play an old animation, it will work and by reverting the order it will select the `PBRAppearance` and ignore the `Appearance` as the parser now only care for the first Appearance it encounters.

Reply:
In that case, we should add a comment in the code about it otherwise, we won't understand what it was done this way.